docker-compose.yml

```diff
@@ -11,11 +11,13 @@ services:
       SEC_CRYPTO_SERVICE_BASE_URI: ${SEC_CRYPTO_SERVICE_BASE_URI}
       SEC_CRYPTO_SERVICE_ENDPOINT_SIGN_PATH: ${SEC_CRYPTO_SERVICE_ENDPOINT_SIGN_PATH}
       SEC_USE_CERTFICATES: ${SEC_USE_CERTFICATES}
-      SEC_KEY_STORE_PATH: ${SEC_KEY_STORE_PATH}
+      #SEC_KEY_STORE_PATH: ${SEC_KEY_STORE_PATH}
       SEC_KEY_STORE_PASSWORD: ${SEC_KEY_STORE_PASSWORD}
     volumes: 

```

Comment:
Do we have sample cert.jks and ca-certificates in the src/main/resources folder? And update readme.md to provide instructions on how to generate those file and update that resources folder? Also for this volume, do we need to update the resources folder permission at the host so that the application is allowed to read the files in the resources folder.

Reply:
The cert.jks is a Java key store file that will change for each deployment and must be kept out of source control for security. The ca-certificates likewise will change per deployment as they are specific for whatever signing authority the deploying party chooses to use, and will also be kept out of source control. The [dev branch README](https://github.com/usdot-jpo-ode/jpo-security-svcs/blob/dev/README.md) already includes detailed description on the JKS file an mentions handling any CA oddities. If we need more detail here, please let us know what additional info you'd like to see.

The changes here in the docker-compose are just more precisely mounting the certificates to proper locations rather than mounting to a generic docker-side volume then copying the files over before running the `update-ca-certificates` - essentially just cutting out a step to simplify the process. As for permissions, we have not ran into permissions issues running this locally, so I don't believe any additional changes are needed.